appkeeper/README.md

```diff
@@ -16,7 +31,7 @@ Get the SIOS AppKeeper API key from AppKeeper GUI.
 
 ![snapshot][1]
```

Comment:
We try to minimize our use of screen shots. If you feel they are absolutely needed, they are fine to keep. But please add alternate text that describes what the screen shows, for people who use screen readers. Something more descriptive than `snapshot`.